src/sentry/migrations/0031_delete_alert_email.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-02-06 21:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def delete_alert_email_user_options(apps, schema_editor):
+    """
+    Processes user reports that are missing event data, and adds the appropriate data
+    if the event exists in Clickhouse.
+    """
+    UserOption = apps.get_model("sentry", "UserOption")
+
+    UserOption.objects.filter(key="alert_email").delete()
```

Comment:
We typically don't do this sort of thing in one giant delete query, since it can potentially run for a long time. We usually do the slower but safer method of using `RangeQuerySetWrapperWithProgressBar` and deleting each item individually. If you do a search you can see a couple of migrations that use it.